cs/modules/ROOT/pages/commands/JsouKolinearni.adoc

```diff
@@ -0,0 +1,26 @@
+= AreCollinear Command
+:page-en: commands/AreCollinear
+ifdef::env-github[:imagesdir: /cs/modules/ROOT/assets/images]
+
+JsouKolinearni( <Bod>, <Bod>, <Bod> )::
+  Ověří, zda jsou body kolineární.
+
+Normálně tento příkaz vypočítá výsledek numericky. Toto chování lze změnit použitím příkazu
+xref:/commands/Prove.adoc[Prove].
+
+[EXAMPLE]
+====
+
+`++JsouKolinearni((1, 2), (3, 4), (5, 6))++` Výsledek je _pravda_, protože všechny tři body leží na stejné přímce..
+
+====
+
+[NOTE]
+====
+
+Podívejte se také na příkazy xref:/commands/AreConcurrent.adoc[AreConcurrent], xref:/commands/AreConcyclic.adoc[AreConcyclic],
```

Comment:
@Struthia please translate the links e.g. `xref:/commands/ProtinajiciSe.adoc[ProtinajiciSe]` -- even if some of those pages are missing for now.